general-cascade/metadata-check.php

```diff
@@ -100,6 +100,8 @@
                 continue;
             }
         }
+        # Please remove any spaces, carriage returns, and non-printing characters, as well as any equal sign, comma, or semi-colon.
+        $key = str_replace(array('/r', '/n', ' ', '=', ',', ';'), '',$key);
```

Comment:
should `/r` and `/n` be changed to `\r` and `\n` for the literal carriage return char and literal newline char?

Reply:
yeah. Apparrently the php online tester i was using wasn't handling the new lines correctly, thus it acted like it was working